.storybook/assets/404.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Page not found</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <link rel="icon" type="image/svg+xml" href="./favicon.svg">
+    <style>
+      body {
+        --spectrum-background-layer-1-color: rgb(253, 253, 253);
+
+        margin: 0;
+        padding: 0;
+        block-size: 100vh;
+        background-color: var(--spectrum-background-layer-1-color);
+      }
```

Comment:
I looked at some of the styles that are being used on our landing page of Storybook as inspiration for this page's styles. I'm totally open to suggestions and changes for the styles. 

Should these be housed somewhere else? Like `base.css` or `index.css` in the `assets` directory? I wanted to try to use some of the typography classes (like `spectrum-Heading` and `spectrum-Body`) but I'm not sure how to link to that CSS.